web/src/scheduled_messages.js

```diff
@@ -319,3 +323,40 @@ export function update_send_later_options() {
         $("#send_later_options").replaceWith(render_send_later_modal_options(filtered_send_opts));
     }
 }
+
+export function show_schedule_message_indicator() {
+    const scheduled_messages = Object.values(scheduled_messages_data);
+
+    const matching_scheduled_messages = scheduled_messages.filter((scheduled_message) => {
+        if (scheduled_message.failed === true) {
+            return false;
+        }
+        switch (scheduled_message.type) {
+            case "private":
+                if (scheduled_message.to.toString() === narrow_state.pm_ids_string()) {
+                    return true;
+                }
```

Comment:
How did you verify that these have the same type? (They would need to be identical algorithms -- e.g. the user IDs sorted ascending with `,` between, but also the same policy about including the current user).

Reply:
When an scheduled message is created, the user IDs are sorted ascending with `,` between.

`pm_ids_string` calls `reply_to_to_user_ids_string` which runs [`sort_numerically`](https://github.com/zulip/zulip/blob/fd221efc68992043e13545e4c9b24c2aa0821593/web/src/people.ts#L342) on the user IDs and sorts them ascending with `,` between.

Reply:
OK -- probably worth writing that down in a comment. Say:

> Both of these will be the user IDs for all participants including the current user sorted in ascending order.